ebr_trackerbot/checker.py

```diff
@@ -0,0 +1,63 @@
+"""
+Checks for failing test from api
+"""
+
+import logging
+import asyncio
+import requests
+import pendulum
+
+
+def check_tests(
+    loop, check_tests_delay, storage, slack_client, api_url, br_url
+):  # pylint: disable-msg=too-many-arguments
+    """
+    Check tests and send slack message to user when some tracked tests failed
+    """
+    logging.info("Check tests results")
+    loop.call_later(check_tests_delay, check_tests, loop, check_tests_delay, storage, slack_client, api_url, br_url)
+    end = pendulum.now("Europe/Berlin")
```

Comment:
Lets default to UTC, and maybe allow the client to specify timezone in their request

Reply:
Alternatively, check to see if there is a config option for the timezone, and if not default to UTC

Reply:
I changed the default to UTC